pkg/vpcmodel/groupingSelfLoop.go

```diff
@@ -121,9 +121,9 @@ func (g *GroupConnLines) findMergeCandidates(groupingSrcOrDst map[string][]*grou
 	for _, key := range relevantKeys {
 		lines := groupingSrcOrDst[key]
 		bucket := lines[0].commonProperties.groupingStrKey
-		subnetIfVsiVPCIfSubnet := g.getSubnetUIDIfVsi(lines[0].src)
+		subnetIfVsiVPCIfSubnet := getSubnetUIDIfVsi(lines[0].src)
 		if subnetIfVsiVPCIfSubnet == "" {
-			subnetIfVsiVPCIfSubnet = g.getVPCUIDIfSubnet(lines[0].src)
+			subnetIfVsiVPCIfSubnet = getVPCUIDIfSubnet(lines[0].src)
```

Comment:
can be replaced by the new function `getSubnetOrVPCUID` ?

Reply:
Not sure I follow; could it be that you are referring to an old version of the code?
This is the current line:
`subnetIfVsiVPCIfSubnet = getVPCUIDIfSubnet(lines[0].src)`

Reply:
why are you using here both `getSubnetUIDIfVsi` and `getVPCUIDIfSubnet` instead of calling `getSubnetOrVPCUID` ?

Reply:
ok, done